### DISCUSSION TITLE
Faces:new

### DISCUSSION THREAD

**Discussion by pittbull** [photoprism/photoprism#2526] [Q&A]
Hi - and thank you so much for creating a fantastic peace of software 😊

I have recently started investigating the face detection parts and I am a bit confused: When going to faces:new and click "show all new faces" I'm getting literally all pictures - from people to trees to coffee cups. I was under the impression this would only list people that aren't matched?

**Comment by lastzero** [comment]
https://docs.photoprism.app/user-guide/organize/people/

**Comment by graciousgrey** [comment]
The faces: new filter only shows photos where PhotoPrism detected faces.  To see the faces you need to open the edit dialog for the photos of the search result.

An option to display faces instead of the whole photo is planned: https://github.com/photoprism/photoprism/issues/1601 :)
